### PR TITLE
mobile friendly redesign and cleaner properly indented code

### PR DESCRIPTION
**Pull request by Nppatel97**
Redesigned the website for mobile devices. Same changes like the last one. Still have some work to do. 4 HTML files still need work. Everything works well but it'll be a good idea to check. Let me know of any changes or errors in code.

### CODE REVIEW

File: schedule_lessons/dashboard/static/dashboard/css/agenda.css [hunk 1]
```diff
@@ -80,10 +91,20 @@ html body {
...
+#notificationsDropdownPane, #myDropdown {
+  position: absolute;
+  text-align: center;
+  margin-top: 10px;
+}
+
+#notificationsDropdownPane{
   height: 400px;
-  width: 500px;
-  overflow-x: hidden;
```
[the remark this answers is not in the document]
Comment (Nppatel97) [reply schedule_lessons/dashboard/static/dashboard/css/agenda.css]: The code that I removed was messing with the responsiveness. I fixed those but don't know how to add notifications. I will change the CSS to make the inside of the notifications look good. And yeah, @AhmedNSidd is right, the biggest change in this PR is making it mobile friendly which sounds huge but all I am doing is changing the styling and positioning, and making the code responsive wherever needed.

**Comment by Nppatel97** [comment]
Sorry on running a little late. I've made another commit that addresses most of the above feedbacks. For the JS files, I think I should work with you guys or one of you should make the changes as I don't want to mess up any code. There may be some little UI issues here and there (like the notification pane mentioned by @AhmedNSidd etc.) and I am working on fixing them. Let me know if I missed something and I'll try my best to get it fixed ASAP. 

Also, at this point, I think this PR for mobile re-design looks clean enough to be pushed to master. The little UI bugs I mentioned are not a big deal and the functionality of any features isn't affected as far as I know. If y'all feel like this PR is ready then feel free to merge it, I'll create another one to make additional changes that I am working on.

File: schedule_lessons/dashboard/static/dashboard/css/agenda.css [hunk 2]
```diff
@@ -28,6 +28,17 @@ html body {
   background-color: #4A4A4A;
 }
 
+#noti-triangle-up, #pro-triangle-up {
+	width: 0;
```
[the remark this answers is not in the document]
Comment (sdevalapurkar) [reply schedule_lessons/dashboard/static/dashboard/css/agenda.css]: i dont see these addressed, would be nice to get them aligned @Nppatel97

File: schedule_lessons/dashboard/static/dashboard/css/agenda.css [hunk 3]
```diff
@@ -29,11 +29,11 @@ html body {
...
 
 #noti-triangle-up, #pro-triangle-up {
-	width: 0;
+  width: 0;
   height: 0;
   float: right;
-	border-left: 9px solid transparent;
-	border-right: 9px solid transparent;
+  border-left: 9px solid transparent;
+  border-right: 9px solid transparent;
```
Comment (sdevalapurkar) [inline schedule_lessons/dashboard/static/dashboard/css/agenda.css]: these changes look good - please make the necessary changes to the similar code present in other files as well 👍

**Comment by Nppatel97** [comment]
I have addressed many of the changes mentioned in this PR. There remain some few though, and I think we should address them in another PR.

**Pull request merged**